.circleci/config.yml

```diff
@@ -117,13 +117,19 @@ commands:
       - run:
           name: Run cucumber scenarios
           command: cd integration_tests && mkdir -p cucumber_output && node_modules/.bin/cucumber-js --tags "not @long-running and not @broken and not @flaky" --format json:cucumber_output/tests.cucumber
+      - run:
+          name: Generate report
+          command: cd integration_tests && node ./generate_report.js
+          when: always
       - run:
           name: Run flaky/broken cucumber scenarios (Always pass)
           command: node_modules/.bin/cucumber-js --tags "not @long-running and @broken" --format json:cucumber_output/broken-tests.cucumber || true
       - store_test_results:
           path: integration_tests/cucumber_output
       - store_artifacts:
           path: integration_tests/cucumber_output
```

Comment:
```suggestion
```

Reply:
If we use `integration_tests/temp/base_nodes` here it will upload all the output test files

Reply:
This one has to be the same as the path in the previous steps

Reply:
I tested `path: integration_tests/temp/base_nodes`; it worked. 
`integration_tests/cucumber_output`  is only for the json output